src/query/src/sql.rs

```diff
@@ -42,10 +42,11 @@ use sql::statements::show::{ShowDatabases, ShowKind, ShowTables};
 use table::requests::{IMMUTABLE_TABLE_LOCATION_KEY, IMMUTABLE_TABLE_PATTERN_KEY};
 use table::TableRef;
 
+use crate::datafusion::execute_show_with_filter;
 use crate::error::{self, Result};
 
 const SCHEMAS_COLUMN: &str = "Schemas";
-const TABLES_COLUMN: &str = "Tables";
+const TABLES_COLUMN: &str = "tables";
```

Comment:
It seems like `arrow-datafusion` doesn't allow use uppercase.

Reply:
What is the influence of this const in this PR 🧐

Reply:
Before this change, we can't run the following code:

```rust

use datafusion::arrow::array::{StringArray};
use datafusion::arrow::datatypes::{DataType, Field, Schema};
use datafusion::arrow::record_batch::RecordBatch;
use datafusion::from_slice::FromSlice;
use datafusion::error::Result;
use datafusion::prelude::*;
use std::sync::Arc;


#[tokio::main]
async fn main() -> Result<()> {
  let schema = Arc::new(Schema::new(vec![
      Field::new("Host", DataType::Utf8, false),
  ]));

  // define data.
  let batch = RecordBatch::try_new(
      schema,
      vec![
          Arc::new(StringArray::from_slice(["127.0.0.1", "0.0.0.0"])),
      ],
  )?;

  let ctx = SessionContext::new();

  ctx.register_batch("foo", batch)?;
  let dataframe = ctx.sql("SELECT Host FROM foo").await?;

  dataframe.show().await?;

  Ok(())
}

```

Generates the following error consistently:

```
Error: SchemaError(FieldNotFound { field: Column { relation: None, name: "host" }, valid_fields: [Column { relation: Some(Bare { table: "foo" }), name: "Host" }] })
```

It seems like `arrow-datafusion` doesn't allow `Host`.

I replaced `Host` with `host`, and then I could run the above code.

Reply:
You can add a double quotes: `"Host"`.

Reply:
@killme2008 

Thanks!!!!!